src/cpp/CMakeLists.txt

```diff
@@ -76,7 +76,29 @@ target_include_directories(${TARGET_NAME_OBJ}
            "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>"
     PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/src")
 
-target_include_directories(${TARGET_NAME_OBJ} SYSTEM PRIVATE "${safetensors.h_SOURCE_DIR}")
+if(ENABLE_GGUF)
+    message(STATUS "Downloading gguflib")
+    FetchContent_Declare(
+      gguflib
+      GIT_REPOSITORY https://github.com/antirez/gguf-tools/
+      GIT_TAG af7d88d808a7608a33723fba067036202910acb3)
```

Comment:
it's better (faster and does not require git) to use archive and SHA256 hash, see an example:
https://github.com/openvinotoolkit/openvino.genai/blob/f1eee93746f8bf80875d4a831a6814a4853bbd3d/src/python/CMakeLists.txt#L7-L11

Reply:
Done